dev/snippets/lib/snippets.dart

```diff
@@ -0,0 +1,13 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+library snippets;
```

Comment:
nit: remove this? (there'll soon be a lint to not name libraries)

Reply:
Done.